app/controllers/organisers_controller.rb

```diff
@@ -4,10 +4,55 @@ class OrganisersController < ApplicationController
   def index
     @organisers = User.where(role: 1)
   end
+  
+  def create
+    @user = User.find_by(email: params[:email])
+    if @user.blank?
```

Comment:
if you add proper validation we won't need this if `if @user.blank?` but just `if @user.save`